commune_app/migrations/0001_initial.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.1.3 on 2022-11-16 09:26
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                ('id', models.IntegerField(primary_key=True, serialize=False, unique=True)),
+                ('full_name', models.CharField(max_length=250)),
```

Comment:
here you have the full name and in the user module you have first_name and last_name separated
it should be the same.